newrelic/resource_newrelic_infra_alert_condition.go

```diff
@@ -60,85 +60,101 @@ func resourceNewRelicInfraAlertCondition() *schema.Resource {
 			},
 			"comparison": {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validation.StringInSlice([]string{"above", "below", "equal"}, false),
+				Description:  "The operator used to evaluate the threshold value. Valid values are above, below, and equal. Supported by the infra_metric and infra_process_running condition types.",
 			},
 			"select": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The attribute name to identify the metric being targeted; for example, cpuPercent, diskFreePercent, or memoryResidentSizeBytes. The underlying API will automatically populate this value for Infrastructure integrations (for example diskFreePercent), so make sure to explicitly include this value to avoid diff issues. Supported by the infra_metric condition type.",
 			},
 			"created_at": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "The timestamp the alert condition was created.",
 			},
 			"updated_at": {
-				Type:     schema.TypeInt,
-				Computed: true,
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: "The timestamp the alert condition was last updated.",
 			},
 			"critical": {
-				Type:     schema.TypeList,
-				MinItems: 1,
-				MaxItems: 1,
-				Optional: true,
-				Elem:     thresholdSchema(),
+				Type:        schema.TypeList,
+				MinItems:    1,
+				MaxItems:    1,
+				Optional:    true,
+				Elem:        thresholdSchema(),
+				Description: "Identifies the threshold parameters for opening a critical alert violation.",
 				//TODO: ValidateFunc from thresholdConditionTypes map
 			},
 			"warning": {
-				Type:     schema.TypeList,
-				Optional: true,
-				MaxItems: 1,
-				MinItems: 1,
-				ForceNew: true,
-				Elem:     thresholdSchema(),
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				MinItems:    1,
+				ForceNew:    true,
+				Elem:        thresholdSchema(),
+				Description: "Identifies the threshold parameters for opening a warning alert violation.",
 				//TODO: ValidateFunc from thresholdConditionTypes map
 			},
 			"integration_provider": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "For alerts on integrations, use this instead of event. Supported by the infra_metric condition type.",
 			},
 			"violation_close_timer": {
 				Type:         schema.TypeInt,
 				Optional:     true,
 				Default:      24,
 				ValidateFunc: intInSlice([]int{0, 1, 2, 4, 8, 12, 24, 48, 72}),
+				Description:  "Determines how much time will pass before a violation is automatically closed. Setting the time limit to 0 prevents a violation from being force-closed.",
```

Comment:
Time in what unit?  Should we list options since it's not a free-form value?

Reply:
👍